"""Error simple

Revision ID: 404339c04245
Revises: a463b3fe4462
Create Date: 2025-08-07 11:51:45.461015

"""
from alembic import op
import sqlalchemy as sa


# revision identifiers, used by Alembic.
revision = '404339c04245'
down_revision = 'a463b3fe4462'
branch_labels = None
depends_on = None


def upgrade():
    # ### commands auto generated by Alembic - please adjust! ###
    with op.batch_alter_table('garments', schema=None) as batch_op:
        batch_op.add_column(sa.Column('order_id', sa.Integer(), nullable=True))
        batch_op.create_foreign_key(None, 'orders', ['order_id'], ['id'])

    # ### end Alembic commands ###


def downgrade():
    # ### commands auto generated by Alembic - please adjust! ###
    with op.batch_alter_table('garments', schema=None) as batch_op:
        batch_op.drop_constraint(None, type_='foreignkey')
        batch_op.drop_column('order_id')

    # ### end Alembic commands ###
